website/docs/integrator-section/verifiable-document/did/signing-document.md

```diff
@@ -22,7 +22,7 @@ In the example above:
 
 ## Verifying the documents
 
-Head to `dev.tradetrust.io` and drag and drop one of the wrapped documents. The document will be verified, then displayed.
+Head to `dev.tradetrust.io` and drag and drop one of the signed documents. The document will be verified, then displayed.
```

Comment:
```suggestion
Head to `dev.tradetrust.io` and drag and drop one of the issued /signed documents. The document will be verified, then displayed.
```

yes right, not wrapped. as document hash needs to be signed by your wallet OR issued to blockchain